### PR TITLE
Fix formatting of Table 3: Escape Sequence Replacements

### PR DESCRIPTION
**Pull request by glyn**

### CODE REVIEW

File: draft-ietf-jsonpath-base.md [hunk 1]
```diff
@@ -717,16 +717,16 @@ in the table below:
...
-| "\" "\"           | U+005C              |
-| "\" "b"           | U+0008              |
-| "\" "f"           | U+000C              |
-| "\" "n"           | U+000A              |
-| "\" "r"           | U+000D              |
-| "\" "t"           | U+0009              |
-| "\" uXXXX         | U+XXXX              |
+| "\\" %x22          | U+0022              |
+| "\\" "'"           | U+0027              |
+| "\\" "/"           | U+002F              |
```
Comment (cabo) [inline draft-ietf-jsonpath-base.md:722]: Almost...

This needs to be "\\" "\\"

(Actually, I'd use `\` in both cases, but this works, too.)

Comment (glyn) [reply draft-ietf-jsonpath-base.md:722]: @cabo Good eyes. I'll fix that later.

Comment (glyn) [reply draft-ietf-jsonpath-base.md:722]: @cabo Now I come to look at this again, I can't see what's wrong - an escaped solidus (`/`) needs to be replaced with a solidus U+002F. What problem did you see?

Comment (cabo) [reply draft-ietf-jsonpath-base.md:722]: The next line has an unescaped `\`.

Comment (glyn) [reply draft-ietf-jsonpath-base.md:722]: @cabo Ah yes! Thanks. BTW PR comments normally refer to the line(s) above.

**Pull request merged**